### ISSUE TITLE
Missing License?

### ISSUE THREAD

**Issue by p-zubieta** [dreid1991/md_engine#30]
I imagine the project was envisioned as being open source as the rest of the MICCoM projects, but there is no License for the project that I can find. Is it just that the License inadvertently missing?

**Comment by jhelfferich** [comment]
Good point. Thanks for bringing this up. I am open for any open source license with a preference for the least restrictive. I think Dan should make the decision which license should apply as he wrote by far most of the code. I am not a lawyer, but I think all contributors need to give their consent to a new license. I will make the start:

I hearby authorize anybody to license or re-license all my past and future contributions to this project under any license that is approved by the Open Source Initiative at the time of licensing or re-licensing.

https://opensource.org/licenses/alphabetical
